lib/lotus/repository.rb

```diff
@@ -674,19 +674,15 @@ def _persisted?(entity)
 
       # Add time create an entity
       def _update_created_at(entity)
-        if entity.class.attributes.include?(:created_at)
+        if entity.respond_to?(:created_at=)
           entity.created_at ||= Time.now.utc
```

Comment:
IMO, attributes in Entity is attr_accessor. So I think I just need to test for a case. I will edit it into `entity.respond_to?(:created_at)` instead of  `entity.respond_to?(:created_at=)`